talk/owt/sdk/include/cpp/owt/conference/remotemixedstream.h

```diff
@@ -11,6 +11,7 @@ namespace conference {
 class RemoteMixedStreamObserver : public owt::base::StreamObserver {
  public:
   virtual void OnVideoLayoutChanged(){};
+  virtual void OnActiveInputChanged(std::string){};
```

Comment:
Prefer to use `const std::string` as the type of first parameter. Please provide a name for this parameter as well. Thanks.

Reply:
Changes done